src/applications/bmqtool/m_bmqtool_interactive.cpp

```diff
@@ -85,6 +87,104 @@ void printMessage(bsl::ostream& out, int index, const bmqa::Message& message)
 #endif
 }
 
+void closeFileStream(bsl::ifstream* fileStream)
+{
+    fileStream->close();
+}
+
+// Load message content from dump file, created by
+// QueueEngineUtil::dumpMessageInTempfile() method. Return `true` on success or
+// `false` otherwise. In case of `false`, `errorDescription` contains details.
+bool loadMessageContentFromFile(
+    bsl::vector<bsl::string>*     payload,
+    bsl::vector<MessageProperty>* messageProperties,
+    bsl::ostream&                 errorDescription,
```

Comment:
In some places, `bsl::string *` is passed to return error messages, here `bsl::ostream`.
Might be good to unify it. I believe passing `bsl::ostream` might be better, since it will be easier to output variable parameters like tokens, strings etc without intermediate `mwcu::MemOutStream`. What do you think?